extension/functions/src/presto.rs

```diff
@@ -34,39 +34,114 @@
 
 use std::sync::Arc;
 
+use arrow::array::*;
+use arrow::error::ArrowError;
 use arrow::{
-    array::{ArrayRef, Time32MillisecondArray},
+    array::ArrayRef,
     datatypes::{DataType, TimeUnit},
 };
-use chrono::Timelike;
+
+use chrono::{Duration, NaiveDate, TimeZone, Utc};
 use datafusion::error::Result;
+use datafusion_common::DataFusionError;
 use datafusion_expr::{
-    ReturnTypeFunction, ScalarFunctionDef, ScalarFunctionPackage, Signature, Volatility,
+    ReturnTypeFunction, ScalarFunctionDef, ScalarFunctionPackage, Signature,
+    TypeSignature, Volatility,
 };
 
 #[derive(Debug)]
-pub struct CurrentTimeFunction;
+pub struct ToIso8601Function;
```

Comment:
@wizardxz I  did not merge origin/release because the requirement for the submission is to only include the new content I wrote. Should the submissions from others also be retained? Perhaps we can communicate about this.